src/components/Crackmapexec/Crackmapexec.tsx

```diff
@@ -99,11 +99,15 @@ const Crackmapexec = () => {
         setLoading(true);
         const args = [];
 
-        if (values.timeout) {
-            args.push(`--timeout ${values.timeout}`);
-        }
-
-        args.push("smb", `${values.ip}`, "-u", `${values.username}`, "-p", `${values.password}`);
+        // No need to check for value since it is mandatory to provide in the portal  
+        args.push("--timeout");
+        args.push(`${values.timeout}`);
+        args.push("smb");
+        args.push(`${values.ip}`);
+        args.push("-u", values.username);
+        args.push("-p", values.password);
+        
+    
```

Comment:
Hi @majore15,

I'm looking at your implementation not its run time which I'll leave for @Phaellynn.
Please run prettier on your code. It is the reason that you're failing CI. This is most likely to the extra lines etc that are not required. Please message me if you need assistance conducting this, however, the following commands should suffice.

**How to fix**

- [ ] Run: yarn run style in the DDT terminal
- [ ] Run: npx prettier --write <filepath> to fix it